### PR TITLE
[SOT] sot export test files

### PR DESCRIPTION
**Pull request by feifei-111**
### PR types

Others
### PR changes

Others
### Description

create test file based on SIR
PCard-66972

origin code like:
image
create test file like:
image

### CODE REVIEW

File: python/paddle/jit/sot/symbolic/export.py [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+
+def export(SIR, path):
+    try:
+        pygen = PyFileGen(SIR)
+        string = pygen.gen_py_codes()
+    except ExportError as e:
+        print("[SOT] Export SIR Failed:", e)
+        return
+
+    with open(path + f"/{SIR.name}.py", "w") as f:
```
Comment (SigureMo) [inline python/paddle/jit/sot/symbolic/export.py]:
```suggestion
    with open(os.path.join(path, f"{SIR.name}.py"), "w") as f:
```

**Review by SigureMo — DISMISSED** [review]
最好加一个简单的单测吧

**Review by Aurelius84 — DISMISSED** [review]
CI 流水线可以找相关同学处理下

**Review by zhangbo9674 — APPROVED** [review]
LGTM for print

File: python/paddle/jit/sot/symbolic/export.py (hunk 1 above)
Comment (feifei-111) [reply python/paddle/jit/sot/symbolic/export.py]: Done

File: test/sot/test_sot_export.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        a = self.linear(x)
+        return a + self.bias
+
+
+class TestSotExport(unittest.TestCase):
+    @with_export_guard("/tmp")
+    def test_basic(self):
+        net = Net()
+        x = paddle.to_tensor([2, 3], dtype="float32", stop_gradient=True)
+        y = paddle.jit.to_static(net)(x)
```
Comment (Aurelius84) [inline test/sot/test_sot_export.py:42]: 按照单测规范，是需要有明确的Assert语句。否则单测是没有意义的

**Review by Aurelius84 — APPROVED** [review]
LGTM and Add UT in this PR

**Pull request merged**